src/assignments/tech-jobs-oo.rst

```diff
@@ -416,20 +554,44 @@ line, make that happen.
    Do NOT add anything beyond what is needed to make the test pass. You will
    add the remaining behaviors for ``ToString()`` as you code each new test.
 
-Finish the TDD for ``ToString()``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
+Test 2: ``TestToStringContainsCorrectLabelsAndData``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #. Code a new test for the second required behavior, then run the tests to make
-   sure the new one fails.
+   sure the new one fails.  Call this test ``TestToStringContainsCorrectLabelsAndData``.
 #. Modify ``ToString()`` to make the new test pass. Also, make sure that your
    updates still pass all of the old tests.
 #. Continue this test-refactor cycle until all of the behaviors we want for
    ``ToString()`` work. Remember to add only ONE new test at a time.
 
+Test 3: ``TestToStringHandlesEmptyField``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#. Code your final test for the last required behavior, 
+   then run the tests to make sure the new one fails. Call this test ``TestToStringHandlesEmptyField``.
+#. Modify ``ToString()`` to make the new test pass.  Also make sure that your updates still pass all the old tests.
+
 Cool! Your ``Job`` class is now complete and operates as desired.
 
-Refactor to DRY the Support Classes
------------------------------------
+Run Your Tests
+^^^^^^^^^^^^^^
+
+Run your ``TestTask5`` unit tests.  Refactor your code as needed.  
+Do not move onto Task 6 until you have passed all of Task 5's auto-grading unit tests.
+
+And don't forget to ``add``, ``commit`` and ``push`` the latest version of your code.
+
+Task 6: Refactor to DRY the Support Classes
+-------------------------------------------
+
+.. admonition:: Note
+
+   This test has been commented out for the rest of the tests to run.
+   
+   Remove the ``/*`` in **line 1** and ``*/`` in **line 96**.
```

Comment:
I think you should clarify this one too! It took me a while to figure out where to un-comment